packages/polaris-viz/src/components/DonutChart/DonutChart.tsx

```diff
@@ -65,6 +71,7 @@ export function DonutChart(props: DonutChartProps) {
         legendPosition={legendPosition}
         renderInnerValueContent={renderInnerValueContent}
         renderLegendContent={renderLegendContent}
+        renderTooltipContent={tooltipOptions ? renderTooltip : undefined}
```

Comment:
Currently, `tooltipOptions` is what enables the tooltip and defines its options in the donut chart. I believe we want the tooltip to be an opt-in experience cc @rachelng. But perhaps another prop like `showTooltip` would make more sense? Interested to see what others think

Reply:
I like the `showTooltip` prop. If we decide to go that route, we should add it to all the other components that can render tooltips.